rotkehlchen/globaldb/cache.py

```diff
@@ -100,15 +121,21 @@ def globaldb_get_general_cache_last_queried_ts_by_key(
     return Timestamp(result[0])
 
 
-def globaldb_delete_general_cache(
+def globaldb_delete_cache(
         write_cursor: DBCursor,
         key_parts: Iterable[Union[str, GeneralCacheType]],
         value: Optional[str] = None,
 ) -> None:
-    """Deletes from cache in globaldb. If value is None deletes all entries with the provided
-    cache key. Otherwise, deletes only single entry `key - value`."""
+    """Deletes from cache in globaldb. In the case of the general cache,
+    If value is None deletes all entries with the provided cache key.
```

Comment:
general cache shouldn't contain anymore multivalues right? it can be deleted from the docstring